simple/rdm_deferred_wq.c

```diff
@@ -507,7 +507,7 @@ static int run_test()
 	}
 	printf(" with counters..."); 
 
-	init_buf_vals(tested_op);
+	init_buf_vals();
```

Comment:
The function can be defined as init_buf_vals(void), to catch such errors at compile time.